docs/util/schema.go

```diff
@@ -30,7 +30,7 @@ type SchemaData struct {
 					Type        string `json:"type"`
 					Description string `json:"description"`
 				} `json:"phone"`
-				URL struct {
+				Url struct {
```

Comment:
Why ? I believe the idiom in Go is to capitalize all the letters in 3 letter acronyms